hack/test-stirunimage.sh

```diff
@@ -127,12 +127,21 @@ test_debug "s2i build with git proto"
 s2i build https://github.com/openshift/cakephp-ex openshift/php-55-centos7 test --run=true --loglevel=5 &> "${WORK_DIR}/s2i-git-proto.log" &
 check_result $? "${WORK_DIR}/s2i-git-proto.log"
 
+if [[ "$OSTYPE" -ne "cygwin" ]]; then
+    test_debug "s2i build with volume options"
+    s2i build https://github.com/openshift/cakephp-ex openshift/php-55-centos7 test --volume "${S2I_WORK_DIR}:/home/:z" &> "${WORK_DIR}/s2i-volume-correct.log"
+    check_result $? "${WORK_DIR}/s2i-volume-correct.log"
+    ! s2i build https://github.com/openshift/cakephp-ex openshift/php-55-centos7 test --volume "${S2I_WORK_DIR}:/home/:z:z" &> "${WORK_DIR}/s2i-volume-incorrect.log"
+    check_result $? "${WORK_DIR}/s2i-volume-incorrect.log"
+fi
```

Comment:
seems like we should have a test that does run on windows too, otherwise --volume isn't getting tested on windows at all.